### ISSUE TITLE
travis fails for maven version 3.3.3

### ISSUE THREAD

**Issue by pieterlukasse** [cBioPortal/cbioportal#1932]
Maven version 3.3.3 does not exist anymore. We need to update the travis.yml to use 3.3.9

See for example 
https://travis-ci.org/thehyve/cbioportal/jobs/178847311

```
--2016-11-25 13:54:47--  http://www.eu.apache.org/dist/maven/maven-3/3.3.3/binaries/apache-maven-3.3.3-bin.tar.gz
Resolving www.eu.apache.org (www.eu.apache.org)... 88.198.26.2, 2a01:4f8:130:2192::2
Connecting to www.eu.apache.org (www.eu.apache.org)|88.198.26.2|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
```

**Comment by pieterlukasse** [comment]
@inodb could you check this? Agree with 3.3.9?

**Comment by inodb** [comment]
@pieterlukasse we could push up the version. So far it was usually just the mirror being down. Usually it works again after a while.

**Comment by pieterlukasse** [comment]
@inodb you mean the mirror only fails for http://www.eu.apache.org/dist/maven/maven-3/3.3.3/binaries/apache-maven-3.3.3-bin.tar.gz and not for http://www.eu.apache.org/dist/maven/maven-3/3.3.9/binaries/apache-maven-3.3.9-bin.tar.gz ?  If you try these 2 links, you will see that only the last one works.

**Comment by inodb** [comment]
@pieterlukasse ah yeah seems like it is still down. We can switch it to `3.3.9`

**Comment by inodb** [comment]
Fixed in #1965
